### PR TITLE
server: add counter for keys locking numbers

### PR DESCRIPTION
**Pull request by cfzjywxk**
### What problem does this PR solve? 

Add lock keys counter metric, [grafana change](https://github.com/pingcap/tidb-ansible/pull/1131)

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

Release note

### CODE REVIEW

File: store/tikv/txn.go [hunk 1]
```diff
@@ -347,8 +348,10 @@ func (txn *tikvTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keysInput
 				}
 			}
 		}
+		if lockCtx.LockKeysCount != nil {
+			*lockCtx.LockKeysCount += int32(len(keys))
```
Comment (coocood) [inline store/tikv/txn.go]: Why not add the metrics directly?

Comment (cfzjywxk) [reply store/tikv/txn.go]: This may be used in slow log or other statistic system tables

**Comment by cfzjywxk** [comment]
@jackysp @coocood  PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**